packages/apps/docs/src/docs/contribute/docs.md

```diff
@@ -24,7 +24,8 @@ from.
 - From GitHub
 - From a Local Environment.
 
-This article goes over each of these options to help you suggest changes to the
+[test link](kadena-dao.md) This article goes over each of these options to help
```

Comment:
is it intended?

Reply:
yes! this way Lisa can work with local markdown files.
during the build these links are fixed to URLs

Reply:
the URL is fine but the link says `test link`

Reply:
a...haha

Reply:
fixed it